test/CMakeLists.txt

```diff
@@ -146,6 +146,15 @@ else ()
   target_compile_definitions(header-only-test PRIVATE FMT_HEADER_ONLY=1)
 endif ()
 
+if (${CMAKE_CXX_STANDARD} GREATER_EQUAL "20")
+  add_fmt_executable(compile-time-formatting-test
+      compile-time-formatting-test.cc test-main.cc)
+  target_link_libraries(compile-time-formatting-test gmock)
+  target_include_directories(compile-time-formatting-test SYSTEM PUBLIC gtest gmock)
+  target_link_libraries(compile-time-formatting-test fmt::fmt-header-only)
+  add_test(NAME compile-time-formatting-test COMMAND compile-time-formatting-test)
+endif ()
```

Comment:
This is not needed, just add new test cases to `compile-test`.

Reply:
Done